scripts/exgefs_atmos_repair_replay.sh

```diff
@@ -0,0 +1,112 @@
+#! /usr/bin/env bash
+
+
+source "${USHgfs}/preamble.sh"
+
+#foutmax=6 #last lead hour to extract 
+cd "${DATA}" || exit 1
+
+#COMIN_03 and {COMIN_ATMOS_MASTER} are directory containing the files that we want to extract
+#Extract c00 files for f003 from PSL reanalysis data
+
+fnh=03
+echo "extracting f${fnh}"
+oufile=${DATA}/gefs.t00z.master.grb2f0${fnh}
+
+infile=${COMIN_03}/GFSPRS.GrbF03
+if [[ -f "${infile}" ]]; then #check if input file exists before extraction
+
+  ${WGRIB2} "${infile}" | grep "TSNOWP" | ${WGRIB2} -i "${infile}" -grib tmp ||true
+  ${WGRIB2} tmp -for "2:2" -append -grib  "${oufile}">/dev/null || true 
+ 
+  ${WGRIB2} "${infile}" | grep ":APCP:surface" | ${WGRIB2} -i "${infile}" -grib tmp ||true
+  ${WGRIB2} tmp -for "1:1" -append -grib  "${oufile}">/dev/null || true
+
+  ${WGRIB2} "${infile}" | grep ":ACPCP:surface" | ${WGRIB2} -i "${infile}" -grib tmp ||true
+  ${WGRIB2} tmp -for "1:1" -append -grib  "${oufile}">/dev/null || true
+
+  ${WGRIB2} "${infile}" | grep ":NCPCP:surface" | ${WGRIB2} -i "${infile}" -grib tmp ||true
+  ${WGRIB2} tmp -for "1:1" -append -grib  "${oufile}">/dev/null || true
+
+  ${WGRIB2} "${infile}" | grep ":HCDC:high cloud layer:0" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null || true
+  ${WGRIB2} "${infile}" | grep ":MCDC:middle cloud layer:0" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null || true
+  ${WGRIB2} "${infile}" | grep ":LCDC:low cloud layer:0" | ${WGRIB2} -i "${infile}" -append -grib "${oufile}">/dev/null || true
+
+  rm tmp
+else
+  echo "${infile} does not exist"
+  export err=1; err_chk
+fi 
+
+export varlist=${varlist_FLXacc03} # Parameter table for f03 acc/ave/min/max variables (PSL data)
```

Comment:
I don't think this needs to be exported.
```suggestion
varlist=${varlist_FLXacc03} # Parameter table for f03 acc/ave/min/max variables (PSL data)
```